setup.cfg

```diff
@@ -47,6 +47,12 @@ doc =
     matplotlib
     seaborn
     statsmodels
+benchmarks =
```

Comment:
Right now you need python>=3.9 to run the benchmark (otherwise I get an error with tracemalloc.reset_...)

Reply:
I've added a note on the readme, but I don't know if it's possible to specific as a config that Python 3.9+ is required.